app/main.py

```diff
@@ -1 +1,41 @@
-# write your code here
+from __future__ import annotations
+
+
+# Fixed class
+class Animal:
+    alive: list = []
```

Comment:
This class attribute should be accessed using the class name, not the instance of the class. It's a good practice to use the class name to access class attributes. So, it would be better to use `Animal.alive` instead of `self.alive`.